docs/reference/modules/cluster/allocation_filtering.asciidoc

```diff
@@ -27,35 +52,8 @@ NOTE: Shards will only be relocated if it is possible to do so without
 breaking another routing constraint, such as never allocating a primary and
 replica shard to the same node.
 
-Cluster-wide shard allocation filtering works in the same way as index-level
-shard allocation filtering (see <<index-modules-allocation>> for details).
```

Comment:
That link should now be in the first paragraph (I don't think it shows in the diff) so still exists on the page.  I was just removing what now became duplicate text through this rearranging.